db/forward_iterator.h

```diff
@@ -86,6 +86,10 @@ class ForwardIterator : public InternalIterator {
  private:
   void Cleanup(bool release_sv);
   void SVCleanup();
+  static void SVCleanup(
```

Comment:
Can you add some documentation here of what these functions do?

Reply:
Added.